### DISCUSSION TITLE
Exporting Whole Library - Can I Exclude Shared Libraries?

### DISCUSSION THREAD

**Discussion by zephyr325** [RhetTbull/osxphotos#915] [General]
Hi!  I'm trying to setup a repeatable big export of all my local photos (I'll probably run this a few times a month) using the following command:

`osxphotos export /Users/MyName/Downloads/Pictures/Export --export-by-date --download-missing --exiftool --person-keyword --album-keyword --sidecar xmp --skip-original-if-edited --touch-file --update --use-photokit`

The command executes correctly, but I've noticed I'm getting several thousand files from all the "Shared Albums" that I'm a part of with other folks.  Is there some programatic way to *just* export the local items, and exclude all those shared albums?  I know I can't use a smart album and just tag that one, and I don't want to have to manually create/recreate a static one that I drag all my Library files into each time I do an export.  Same goes for manually

Thanks!

**Comment by RhetTbull** [comment]
Yes, add `--not-shared`  to the command line.

```
--not-shared          Search for photos not in shared iCloud album (Photos
                      5 only).
```

**Reply by RhetTbull** [reply]
And I need to update the help text as it's "Photos 5+"

**Comment by zephyr325** [comment]
Great, thanks!  I missed that in the documentation.

**Comment by oPromessa** [comment]
Hey @zephyr325 , a config I also use to avoid exporting the same pic when it shows in multiple albums (namely Smart Albums) is explained in #610 "How does --folder option work when the same pic is in and out of the specified folder?" on the following https://github.com/RhetTbull/osxphotos/discussions/610#discussioncomment-3494304

Hope it helps!
